modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/impl/ItMetaStorageWatchTest.java

```diff
@@ -218,6 +219,12 @@ private void startCluster(int size) throws NodeStoppingException {
         String name = nodes.get(0).name();
 
         nodes.get(0).cmgManager.initCluster(List.of(name), List.of(name), "test");
+
+        CompletableFuture<Void> startFut = allOf(
```

Comment:
Why not simply make a loop? You don't have to create new future. But that's just a question

Reply:
True